lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -79,7 +79,7 @@ const PolygonParameterEditor = React.createClass({
             points.push(CesiumMath.toDegrees(cartographic.latitude));
             pointsLongLats.push(points);
         }
-        this.props.parameterValues[this.props.parameter.id] = pointsLongLats;
+        this.props.parameterValues[this.props.parameter.id] = [pointsLongLats];
```

Comment:
But it seems all *ParameterEditor.jsx classes do this? E.g. https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Analytics/DateTimeParameterEditor.jsx#L43
https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Analytics/PointParameterEditor.jsx#L49
https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Analytics/RegionParameterEditor.jsx#L97

I wrote PolygonParameterEditor based on PointParameterEditor. If setting props is wrong (and it seems like it is) then I think we will have to change this in all *ParameterEditor.jsx classes?